src/category_theory/limits/limits.lean

```diff
@@ -403,7 +403,7 @@ section limit
 /-- `has_limit F` represents a particular chosen limit of the diagram `F`. -/
 class has_limit (F : J ⥤ C) :=
 (cone : cone F)
-(is_limit : is_limit cone . tactic.apply_instance)
+(is_limit : is_limit cone)
```

Comment:
Why are you changing these lines? I such a change would require at least some comments, and probably some discussion. In general we've making heavy use of implicit tactics in the category theory library.
cc @semorrison @rwbarton

Reply:
This change was made by @semorrison in 0b5f3cc5976cb340f94b6c725f21c513554828ec. I do not know why it was made.

Reply:
Aha, I see. @semorrison Do you by chance remember why you made this change?

Reply:
`is_limit` isn't a class, so `apply_instance` isn't going to have any traction.

Reply:
Ooh rigth. Thanks for explaining!